package/MDAnalysis/core/selection.py

```diff
@@ -963,14 +963,47 @@ class PropertySelection(Selection):
     ])
 
     def __init__(self, parser, tokens):
+        """
+        Possible splitting around operator:
+
+        prop x < 5
+        prop x< 5
+        prop x <5
+        prop x<5
+        """
         prop = tokens.popleft()
+        oper = None
+        value = None
         if prop == "abs":
             self.absolute = True
             prop = tokens.popleft()
         else:
             self.absolute = False
-        oper = tokens.popleft()
-        self.value = float(tokens.popleft())
+
+        # check if prop has any extra information atm
+        for possible in ('<=', '>=', '==', '!=', '<', '>'):
+            try:
+                x, y = prop.split(possible)
+            except ValueError:
+                pass
+            else:
+                prop = x
+                oper = possible + y  # add back after splitting
+                break
+
+        if oper is None:
+            oper = tokens.popleft()
+        # check if oper has the value appended
+        for possible in ('<=', '>=', '==', '!=', '<', '>'):
```

Comment:
The tuple of operators could go in a separate variable to avoid being repeated and ease the addition of new operators.